### DISCUSSION TITLE
Can I generate/get backup phrase in code instead of getting it from email?

### DISCUSSION THREAD

**Discussion by rinzler99** [Web3Auth/Web3Auth#447] [Help]
I want to get the backup/seed phrase in code instead of receiving it in email. Is this possible? if yes please provide steps.

**Comment by AndonMitev** [comment]
yes please i'm looking for this as well

**Comment by chaitanyapotti** [comment]
we return a backup share for custom verifiers back to the application (dappShare) which you can store as a backup, plz go through this answer for more information about dappShare : https://github.com/orgs/Web3Auth/discussions/427#discussioncomment-3442052
